src/main/java/de/retest/recheck/ioerror/ReTestIOException.java

```diff
@@ -8,10 +8,20 @@
 	private static final long serialVersionUID = 1L;
 	private final URI location;
 
+	public ReTestIOException( final URI location ) {
```

Comment:
If the URI-only ctors (also in subclasses) isn't effectively used, I wouldn't add it just for the sake of completeness.